package.json

```diff
@@ -28,27 +28,33 @@
   "author": "kamiazya <yuki@kamiazya.tech>",
   "type": "module",
   "scripts": {
-    "build": "turbo run build",
-    "doc": "turbo run build && typedoc",
     "lint": "biome check --apply .",
-    "test": "turbo run test:unit test:integration",
-    "test:unit": "vitest run"
+    "build": "pnpm -r codegen && pnpm -r build",
+    "doc": "pnpm -r codegen && typedoc",
+    "test": "pnpm -r codegen && vitest",
+    "test:integration": "echo 'No integration tests' && exit 1"
```

Comment:
The coupling test will be implemented in the future, and at this stage the status code is 1 because it is ng for the required quality of the project.

It should end with 0 when implemented in the future.